src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -100,6 +105,22 @@ public static ShaderPipeline getPipeline() {
 		return pipeline;
 	}
 
+	public static void reload() throws IOException {
+		//currently this first line can be used to reload to a diff shaderpack, but it should be removed
+		//when there is a gui or a better system for changing packs at runtime
+		//or could be kept for the gui to use
+		irisConfig.initialize();
+		Path shaderpacksDirectory = FabricLoader.getInstance().getGameDir().resolve("shaderpacks");
+		Path shaderPackRoot = shaderpacksDirectory.resolve(irisConfig.getShaderPackName());
+		Path shaderPackPath = shaderPackRoot.resolve("shaders");
```

Comment:
It feels like there's probably some code duplication going on here with the main Iris initialization routines